mongocli.md

```diff
@@ -0,0 +1,93 @@
+# MongoDB CLI
+
+The MongoDB CLI is a modern command line interface that enables you to manage your MongoDB services from the terminal.
+
+![mongocli-atlas-quickstart](https://user-images.githubusercontent.com/461027/126986233-0dd5c82a-2c75-4887-ab66-eb018c59e093.gif)
+
+Use simple, one-line commands to interact with MongoDB Atlas, Cloud Manager, or Ops Manager, and to automate management tasks for your deployments.
+
+## Documentation
+
+See the [official docs](https://docs.mongodb.com/mongocli/stable/) for instructions on how to
+install, set up, and reference available commands.
+
+## Installing
+
+### Homebrew on macOS
+
+```bash
+brew install mongocli
+```
+
+### Pre-built Binaries
+
+Download the appropriate version for your platform from [mongocli releases](https://github.com/mongodb/mongocli/releases).
+After you download the library, you can run it from anywhere and don't need to install it into a global location.
+This works well for shared hosts and other systems where you don't have a privileged account.
+
+Ideally, you should place this binary somewhere in your `PATH` for ease of use. 
+`/usr/local/bin` is the most probable location.
+
+### Build From Source
+
+#### Fetch Source
+
+```bash
+git clone https://github.com/mongodb/mongocli.git
+cd mongocli
+```
+
+#### Build
+
+To build `mongocli`, run:
+
+```bash
+make build
+```
+
+The resulting `mongocli` binary is placed in `./bin`.
+
+#### Install
+
+To install the `mongocli` binary in `$GOPATH/bin`, run:
+
+```bash
+make install
+```
+
+**Note:** running `make build` is not needed when running `make install`.
+
+## Usage
+
+To get a list of available commands, run `mongocli help`
+or check our [online documentation](https://docs.mongodb.com/mongocli/master/) for more details.
```

Comment:
For consistency, suggest: "or check our [documentation](https://docs.mongodb.com/mongocli/master/) for more details."